io_scene_nif/modules/nif_export/animation/transform.py

```diff
@@ -168,8 +168,8 @@ def export_kf_root(self, b_armature=None):
             # if variable_2:
             # controlledblock.set_variable_2(variable_2)
         else:
-            raise util_math.NifError("Keyframe export for '%s' is not supported.\nOnly Morrowind, Oblivion, Fallout 3, Civilization IV,"
-                                     " Zoo Tycoon 2, Freedom Force, and Freedom Force vs. the 3rd Reich keyframes are supported." % bpy.context.scene.niftools_scene.game)
+            raise util_math.NifError(f"Keyframe export for '{bpy.context.scene.niftools_scene.game}' is not supported.\nOnly Morrowind, Oblivion, Fallout 3, Civilization IV,"
```

Comment:
I wonder what multi-line f-string would do to the output. Could get rid of the \n chars.

Reply:
Do you need to put an f on the second line?

Reply:
Yes, the second `f` string is required. It'll error out, otherwise.
Works:
```
print(f'something\n'
      f'something, two.')
```
Doesn't work (syntax error).
```
print(f'something\n'
       'something, two.')
```